packages/postcss-lowercase-text/src/__tests__/atRules.test.js

```diff
@@ -0,0 +1,120 @@
+import postcss from 'postcss';
+
+import plugin from '..';
+
+function run(testMsg = 'should work', input, output) {
+  it(testMsg, (done) => {
+    let result = postcss([plugin]).process(input, {
+      from: undefined,
+    });
+    expect(result.css).toEqual(output);
+    done();
+  });
+}
+
+describe('transforming @media', () => {
+  run(
+    'should safely transform the @media and its properties',
+    '@MEDIA screen and (min-width: 480px){body{COLOR: lightgreen;}}',
+    '@media screen and (min-width: 480px){body{color: lightgreen;}}'
+  );
+
+  run(
+    'should safely transform the @media but not the params',
+    '@MEDIA SCREEN and (min-width: 480px){}',
+    '@media screen and (min-width: 480px){}'
+  );
+});
```

Comment:
Please add test: `@MEDIA SCREEN AND (MIN-WIDTH: 480PX) {}`

Reply:
ok cool 👍